src/ZeroDowntimeEventReplaysServiceProvider.php

```diff
@@ -35,6 +35,6 @@ public function register()
             return new RedisReplayRepository();
         });
 
-        return parent::register(); // TODO: Change the autogenerated stub
+        parent::register(); // TODO: Change the autogenerated stub
```

Comment:
`register()` should not return anything.

There is an error in `./vendor/spatie/laravel-package-tools/src/PackageServiceProvider.php` too.